docs/source/reference/api_reference.md

```diff
@@ -137,13 +138,13 @@ for message in chat_completion:
 
 ## Cloud Providers
 
-TGI can be deployed on various cloud providers for scalable and robust text generation. One such provider is Amazon SageMaker, which has recently added support for TGI. Here's how you can deploy TGI on Amazon SageMaker:
+TGI can be deployed on various cloud providers for scalable and robust text generation. Among those cloud providers, both Amazon Sagemaker and Google Cloud have TGI integrations within their cloud offering.
 
-## Amazon SageMaker
+### Amazon SageMaker
 
 To enable the Messages API in Amazon SageMaker you need to set the environment variable `MESSAGES_API_ENABLED=true`.
 
-This will modify the `/invocations` route to accept Messages dictonaries consisting out of role and content. See the example below on how to deploy Llama with the new Messages API.
+This will modify the `/invocations` route to accept Messages dictionaries consisting out of role and content. See the example below on how to deploy Llama with the new Messages API.
```

Comment:
I would say:

> This will modify the `/invocations` route to accept [Messages dictionaries](#openai-messages-api). See the example below on how to deploy Llama with the new Messages API.

Reply:
this is subject to change, check main branch! now there will be no need for MESSAGES API ENABLED configuration parameter to use chat template or not